Makefile

```diff
@@ -1,8 +1,11 @@
 .PHONY: default all requirements configure repo_sync ironic build ocp_run clean ocp_cleanup host_cleanup
-default: requirements configure repo_sync ironic build ocp_run
+default: prereq requirements configure repo_sync ironic build ocp_run
 
 all: default
 
+prereq:
+	./00_prerrequisites.sh
```

Comment:
CI failed because of a extra "r" here

Reply:
My bad, fixed. In Spanish we double 'r' if surrounded by vowels depending on the sound (like in 'perro').